requirements.txt

```diff
@@ -1,7 +0,0 @@
-appdirs~=1.4.3
```

Comment:
Why was `requirements.txt` deleted?

Reply:
It conflicted with requirements in setup.py and isn't used anywhere.

Reply:
An advertisement for my project pypi-mobans: requirements of mobanized project are governed centrally by one single source of truth: project-name.yaml. Change in one place, everywhere else follows.

Reply:
Isn't that what `install_requires` in setup.cfg is for?